docs/GUIDE.en.md

```diff
@@ -1249,6 +1250,132 @@ near account \
 </a>
 </details>
 
+#### get-public-key - Get the public key to your account
+
+- [from-ledger](#from-ledger---Get-the-public-key-stored-on-your-Ledger-Nano-device)
+- [from-seed-phrase](#from-seed-phrase---Get-the-public-key-with-the-seed-phrase)
+- [from-plaintext-private-key](#from-plaintext-private-key---Get-the-public-key-from-the-plaintext-private-key)
+- [from-keychain](#from-keychain---Get-the-public-key-stored-in-a-secure-keychain)
+- [from-legacy-keychain](#from-legacy-keychain---Get-the-public-key-stored-in-the-legacy-keychain-compatible-with-the-old-near-CLI)
+
+#### from-ledger - Get the public key stored on your Ledger Nano device
+
+To get the public key from Ledger, enter in the terminal command line:
+```txt
+near account \
+    get-public-key \
+    from-ledger --seed-phrase-hd-path 'm/44'\''/397'\''/0'\''/0'\''/1'\'''
```

Comment:
nit: this could use non machine-generated form with double quotes `"m/44'/397'/0'/0'/1'"` , which is better readable as an example

Reply:
it's the same nit in the spot with `from-seed-phrase`

Reply:
Done